File: Preproc/create_train_test.pl
#!/usr/bin/perl -w

use strict;

use vars qw ();

my $data_d = "/home/jibmaird/Data/Corpora/mutation-corpus-master/mutation-corpus-master/main/merged-corr";

my $out_d = "/home/jibmaird/Data/Experiments/Mutation/1";

for(my $i=0;$i<=7;$i++) {

    opendir(D,"$data_d/0$i")||die;
    my @File = grep /\d/,readdir D;
    closedir(D);

    foreach my $f (@File) {
	if ($i < 6) {
	    system("cp $data_d/0$i/$f $out_d/Train");
	}
	else {
	    system("cp $data_d/0$i/$f $out_d/Test");
	}
    }
}

File: Preproc_py/create_train_test.py
import os
from shutil import copy

data_d = "/Users/david/Projects/Data/Corpora/amia-18-mutation-corpus/data";
out_d = "/Users/david/Projects/Data/Experiments/Mutation/1";

if not os.path.isdir(out_d):
    os.mkdir("/Users/david/Projects/Data/Experiments/Mutation")
    os.mkdir(out_d)
    os.mkdir(out_d+"/Train")
    os.mkdir(out_d+"/Test")
    
for i in range(0,7):
    for file in os.listdir(data_d+"/0"+str(i)):
      if i < 6:
        copy(data_d+"/0"+str(i)+"/"+file,out_d+"/Train")
      else:
        copy(data_d+"/0"+str(i)+"/"+file,out_d+"/Test")
    
